GTA V Script Decompiler/Utils.cs

```diff
@@ -23,63 +24,97 @@ public static uint jenkins_one_at_a_time_hash(string str)
 			hash += (hash << 15);
 			return hash;
 		}
-		public static string formathexhash(uint hash)
+		public static string FormatHexHash(uint hash)
 		{
-			string hashres = hash.ToString("X");
-			while (hashres.Length < 8)
-				hashres = "0" + hashres;
-			return hashres;
+			return $"0x{hash:X8}";
 		}
 		public static float SwapEndian(float num)
 		{
+            var b = BitConverter.GetBytes(num);
+
+            return (float)((b[3] << 0) | (b[2] << 8) | (b[1] << 16) | (b[0] << 24));
```

Comment:
this wont return the correct value(like how the array method worked), instead it will return a floating point representation of the bitshifted value as a uint